packages/common-styles/just.config.ts

```diff
@@ -0,0 +1,3 @@
+const { preset } = require('@uifabric/build');
+
+preset();
```

Comment:
You can simplify this:
```suggestion
const { preset, just } = require('@uifabric/build');
const { task, series } = just;

preset();

task('build', series('clean', 'copy')).cached();
```